src/dune_rules/watermarks.ml

```diff
@@ -239,8 +239,9 @@ let make_watermark_map ~commit ~version ~dune_project ~package =
     | Some value -> Ok (String.concat ~sep value)
   in
   let make_dev_repo_value = function
-    | Some (Package.Source_kind.Github (user, repo)) ->
-      Ok (sprintf "https://github.com/%s/%s" user repo)
+    | Some (Package.Source_kind.Host h) ->
+      (* Is this right? shouldn't this include git+https? *)
```

Comment:
I think `git+https` is only necessary for `opam pin` when `-k git` is not passed. `git clone` works fine with just `https`. So it really depends what `%PKG_REPO%` is supposed to be. It's described as "the value of the repo field", but there's no repo field in opam. There's dev-repo, but it includes `git+https` in `dune.opam`, so it must be filled through another path. So I'm confused as well :)

Reply:
I see. That makes sense. I'll remove the comment then.